Description: fix compatibility with lz4 1.9.2 which prints the banner to
 stdout instead of stderr
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/mysql-8.0/+bug/1851696

--- a/mysql-test/t/mysqlpump_basic_lz4.test
+++ b/mysql-test/t/mysqlpump_basic_lz4.test
@@ -1,7 +1,7 @@
 if (!$LZ4_DECOMPRESS) {
   let LZ4_EXEC_LOG= $MYSQLTEST_VARDIR/log/lz4_exec_log.txt;
   --error 0, 1, 127
-  --exec lz4 -V 2> $LZ4_EXEC_LOG
+  --exec lz4 -V > $LZ4_EXEC_LOG 2>&1
   let LZ4_STATUS= $__error;
   --error 0,1
   --remove_file $LZ4_EXEC_LOG
